client/src/app/components/contact-us/contact-us.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="op-contact-us">
+  <h2>יצירת קשר</h2>
+  טקסט לדוגמא על יצירת קשר
+  <form (submit)="submitMe()">
+    <div>
+      <input type="text" name="contactName" [(ngModel)]="contactName" placeholder="שם"/>
```

Comment:
I don't think we need two way data binding (banana in a box). We don't change the values of the `input` in our code, we just take this value and send it in a mail, one way data binding is better.